client/my-sites/stats/stats-redirect/index.tsx

```diff
@@ -1,35 +1,14 @@
-import config from '@automattic/calypso-config';
```

Comment:
It feels the whole stats-redirect module could be removed, couldn't it?

Reply:
Good catch! Indeed, the `LoadStatsPage` component has already checked the user with authority.

Reply:
Addressed in [0d909a1](https://github.com/Automattic/wp-calypso/pull/95436/commits/0d909a1eecb5c49e8ced194de7974905dc9cb72e).